deploy/kubernetes/helm/che/values.yaml

```diff
@@ -37,3 +37,4 @@ global:
   gitHubClientID: ""
   gitHubClientSecret: ""
   pvcClaim: "1Gi"
+  cheInfraKubernetesNamespace: ""
```

Comment:
Maybe it would be better to name it something like `cheWorkspacesNamespace` from helm chart perspective